src/main/webapp/app/entities/flair-bi/filter/filter-parameters.service.js

```diff
@@ -481,7 +481,8 @@
                                 }
                             }
                         });
-                        filterUrl = element.properties[0].value + "&filters=" + JSON.stringify(filterUrl);
+                        var url = removeURLParameter(element.properties[0].value, "filters");
+                        filterUrl = url + "&filters=" + JSON.stringify(filterUrl);
```

Comment:
Create a new function to build filter url

Reply:
sure